bin/ramalama

```diff
@@ -8,7 +8,7 @@
 
 
 def main(args):
-    sharedirs = ["/opt/homebrew/share/ramalama", "/usr/local/share/ramalama", "/usr/share/ramalama"]
+    sharedirs = ["./", "/opt/homebrew/share/ramalama", "/usr/local/share/ramalama", "/usr/share/ramalama"]
```

Comment:
I'm not sure this will work on an installed system. The way this code works is, go through the list one by one and select the first directory that exists.

"./" will always exists.

I think this would work better:

```
sharedirs = ["ramalama", "/opt/homebrew/share/ramalama", "/usr/local/share/ramalama", "/usr/share/ramalama"]
```

This way if we run from the base of the git repo, it will use "ramalama" as a syspath.

Reply:
I changed it to look for the subdir ramalama, although